rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/KafkaInstanceAdminClientImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.redhat.service.bridge.rhoas;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.openshift.cloud.api.kas.auth.models.AclBinding;
+import com.openshift.cloud.api.kas.auth.models.AclOperationFilter;
+import com.openshift.cloud.api.kas.auth.models.AclPatternTypeFilter;
+import com.openshift.cloud.api.kas.auth.models.AclPermissionTypeFilter;
+import com.openshift.cloud.api.kas.auth.models.AclResourceTypeFilter;
+import com.openshift.cloud.api.kas.auth.models.NewTopicInput;
+import com.openshift.cloud.api.kas.auth.models.Topic;
+
+import io.quarkus.oidc.client.NamedOidcClient;
+import io.quarkus.oidc.client.Tokens;
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.Vertx;
+
+@ApplicationScoped
+public class KafkaInstanceAdminClientImpl extends AbstractAppServicesClientImpl implements KafkaInstanceAdminClient {
+
+    @ConfigProperty(name = "event-bridge.rhoas.instance-api.host")
+    String basePath;
+
+    @Inject
+    Vertx vertx;
+
+    @Inject
+    @NamedOidcClient("mas-sso")
+    Tokens tokens;
```

Comment:
We probably need to review token refresh mechanism. I think it is better to do that with another PR

Reply:
I was checking what @r00ta is doing in #198 , but probably we can move on with this and then refactor the token refresh mechanism later once we finalize it.

Reply:
Let's refactor this in a dedicated PR once we agree on a final refresh mechanism to overcome Quarkus issues.